src/content/structured/components/chips/code.mdx

```diff
@@ -187,16 +187,21 @@ export const snippetsOutline = [
 export const snippetsDismissible = [
   {
     language: "Web component",
-    snippet: `<ic-chip label="Default" dismissible="true">
+    snippet: `<ic-chip label="Default" dismissible="true" id='my-chip'>
   <svg slot="icon" ... />
 </ic-chip>
 <ic-chip label="Default" appearance="outline" dismissible="true">
   <svg slot="icon" ... />
-</ic-chip>`,
+</ic-chip>
+<script>
+  const chip = document.querySelector('#my-chip');
+  chip.addEventListener('icDismiss', (ev) => console.log(ev));
+</script>
+`,
   },
   {
     language: "React",
-    snippet: `<IcChip label="Default" dismissible>
+    snippet: `<IcChip label="Default" dismissible onIcDismiss={(ev) => console.log(ev)}>
```

Comment:
need to add this into the ComponentPreview?

Reply:
Updated. This will also require a component library release due to https://github.com/mi6/ic-ui-kit/pull/347